src/strategies/otterspace-badges/index.ts

```diff
@@ -0,0 +1,81 @@
+import { subgraphRequest } from '../../utils';
+import examplesFile from './examples.json';
+
+export const author = 'otterspace';
+export const version = '1.0.0';
+export const examples = examplesFile;
+
+const OTTERSPACE_SUBGRAPH_API_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/otterspace-xyz/badges-optimism',
+  '5': 'https://api.thegraph.com/subgraphs/name/otterspace-xyz/badges-goerli'
+};
+
+const params = {
+  badges: {
+    __args: {
+      where: {
+        spec_: {
+          raft: ''
+        }
+      }
+    },
+    owner: true,
+    spec: {
+      id: true
+    }
+  }
+};
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  snapshot
+) {
+  // Get all the badges for a raft
```

Comment:
- comment _"Get all the badges for a raft"_ could be a file-level function `fetchBadgesForRaft(network: string, raftAddress: string, raftTokenId: string)`
- mutating a file-level constant is unusual; we could instead use a function, e.g.  `const params = buildSubgraphQueryParams(options.raftAddress, options.raftTokenId)`
- who ensures `network` is a valid key for `OTTERSPACE_SUBGRAPH_API_URL`? should we handle errors explicitly?
- same question for potential failure of `subgraphRequest()`; assumption: letting thrown error uncaught is fine